app/Http/Transformers/PostTransformer.php

```diff
@@ -30,12 +30,15 @@ public function transform(Post $post)
             'signup_id' => $post->signup_id,
             'northstar_id' => $post->northstar_id,
             'media' => [
-
                 'url' => $post->getMediaUrl(),
                 'original_image_url' => $post->url,
                 'caption' => $post->caption,
             ],
             'tags' => $post->tagSlugs(),
+            'reactions' => [
+                'reacted' => $post->reactions->isNotEmpty() ? true : false,
```

Comment:
I think this ternary can probably be removed if `isNotEmpty` already returns a boolean. Also, do we eager-load reactions everywhere this transformer is used? If not, this might cause some unexpected performance issues.

Reply:
@DFurnes is this using the eager-loaded object or just querying the post/reaction relationship?

Reply:
It depends where it's being used! I'm not sure if there's a good way to check this at runtime, so we might want to just do a quick audit of everywhere `PostTransformer` gets applied and check.

Reply:
```
$query = $query->with(['reactions' => function ($query) use ($userId) {
    $query->where('northstar_id', '=', $userId);
}]);
```
conditionally adds `reactions` to the query if `as_user` is included, so we are just checking to see if that pulled up any reactions or not.

Reply:
Oh right, got it! Hmm and just to make sure this wouldn't load the overall reactions relationship otherwise right?

Reply:
A-ha yeah it was loading it here if it didn't have it already. Needed to add `$post->relationLoaded('reactions')`